tests/test_importers.py

```diff
@@ -136,6 +140,9 @@ def test_archiving_files(self):
         assert "sen_tracks.rep" in names
         assert "uk_track.rep" in names
 
+        # Assert that there is no file in the input_files folder anymore
```

Comment:
Hmm, I think this is checking that the target folder has been cleaned up.

I think we still need to test that the input files get moved on successful import. We're testing that they've arrived at the new location, we just aren't testing that they're no longer present in the start/original location.

Reply:
Done:
https://github.com/debrief/pepys-import/pull/216/commits/28b30864dba0073933e64bea33110b34b0a2ff6b